langchain/tools/python/tool.py

```diff
@@ -18,7 +18,7 @@
 
 
 def _get_default_python_repl() -> PythonREPL:
-    return PythonREPL(_globals=globals(), _locals=None)
+    return PythonREPL(_globals=None, _locals=None)
```

Comment:
why?

Reply:
The goal of this PR is to offer better security in use of Python REPL, and passing in `globals()` looks like a massive security hole to me. I see a lot of code out there where people expose e.g. their API keys and other secrets as globals in memory space, and those could be a prompt injection target. The rest of your comments, @hwchase17 seem to be towards making a higher security level optional, and I'd reluctantly accept that (for me, higher security should be the default), but if the user dos opt for a tighter ship, then that `_globals=None` is IMO an important part of the picture.

Reply:
agreed with @uogbuji but also right now the Wasm interpreter doesn't supported globals that can't be represented as a type that can be easily converted to string. Often the result of the globals function call are object types that can't be converted to string.